presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -49,6 +51,16 @@ public void testAggregateSingleColumn()
         assertQuery("SELECT MAX(comment) FROM orders");
     }
 
+    protected boolean supportsDefaultValues()
+    {
+        return false;
```

Comment:
Why do we need two methods? Maybe createTableWithDefaultColumns could throw `SkipException`?